eqs/src/configuration.rs

```diff
@@ -44,7 +70,7 @@ pub struct EQSOptions {
 fn default_data_path() -> PathBuf {
     let mut data_dir = data_local_dir()
```

Comment:
I'll resolve this; since the wallet and EQS use storage for very different purposes (user-facing files vs internal-only persistent state) I think it's fine for the wallet to use `~` and the EQS to use `data_local_dir`, and then it doesn't matter if they have the same path or not